pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+# Overview
+
+*Provide a general overview of how your code contribution works, detailing the
+effects and consequences of the change.*
+
+# Current behaviour
+
+*Describe the current behaviour of the project and how it differs from the
+behaviour your change is proposing.*
+
+# Expected behaviour
+
+*Describe the behaviour your change is proposing to the project. If your change
```

Comment:
These two fields blend into other - the current behaviour talks about how it differs - maybe that part should be dropped.
Also i think Expected is wrong - Describe the behaviour change may be a more accurate title.

Reply:
I've updated the sections now, does this look clearer?